keras_cv/backend/__init__.py

```diff
@@ -13,17 +13,52 @@
 # limitations under the License.
 from keras_cv.backend.config import multi_backend
 
+# Keys are of the form: "module.where.attr.exists->module.where.to.alias"
+# Value are of the form: ["attr1", "attr2", ...] or
+#                        [("attr1_original_name", "attr1_alias_name")]
+_KERAS_CORE_ALIASES = {
+    "saving->utils": [
+        "register_keras_serializable",
```

Comment:
Is this not accessible under `keras.saving` for both?

Reply:
I did this to avoid having to change `keras.utils` to `keras.saving`. Just makes it much faster to port models.